src/docs.js

```diff
@@ -1,4 +1,35 @@
-function documenter() {
+let tar = require('tar-stream');
```

Comment:
This will need to be added to `package.json`. You should be able to do `npm install --save tar-stream`.